tests/numpy/common.py

```diff
@@ -138,7 +138,10 @@ def get_rand_arr(ddesc):
                     reference_result = [reference_result]
                     dace_result = [dace_result]
                     for ref, val in zip(reference_result, dace_result):
-                        assert np.allclose(ref, val, equal_nan=True)
+                        if ref.dtype == np.float32:
+                            assert np.allclose(ref, val, equal_nan=True, rtol=1e-3, atol=1e-5)
```

Comment:
Aren't those error bounds a bit too loose?

Reply:
I took it from another test I had, anyway they were too tight with 1e-8 atol.